install.py

```diff
@@ -242,9 +236,8 @@ def main():
         sys.exit(1)
 
     for project in ["pyobjc-core"] + sorted_framework_wrappers():
-        ok = build_project(project, sys.argv[1:])
-        if not ok:
-            break
+        print(f"\nBuilding {project}...\n")
+        build_project(project, sys.argv[1:])
```

Comment:
Not ok. The old code is intentional, the code in framework wrappers should either build correctly or be skipped entirely (that is, bindings for frameworks introduced in macOS 11 will be skipped automatically on macOS 10.15 or earlier).